packages/simcore-sdk/tests/integration/test_node_ports_v2_nodeports2.py

```diff
@@ -21,6 +22,10 @@
 from simcore_sdk.node_ports_v2.links import ItemConcreteValue
 from simcore_sdk.node_ports_v2.nodeports_v2 import Nodeports
 
+SYMLINK_PATH = Path(tempfile.gettempdir()) / f"symlink_{Path(__file__).name}"
+if not SYMLINK_PATH.exists():
+    os.symlink(__file__, SYMLINK_PATH)
```

Comment:
I might be missing something here, but are you creating a symlink each time that module is read?
-> fixture with cleanup code ???

Reply:
migrated to a fixture